app/src/main/java/com/compscieddy/meetinthemiddle/GroupAdapter.java

```diff
@@ -0,0 +1,53 @@
+package com.compscieddy.meetinthemiddle;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import de.hdodenhof.circleimageview.CircleImageView;
+
+/**
+ * Created by ambar on 6/7/16.
+ */
+public class GroupAdapter extends RecyclerView.Adapter<GroupAdapter.GroupHolder> {
+
+    private Context mContext;
+
+    @Override
+    public GroupAdapter.GroupHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        mContext = parent.getContext();
+        LayoutInflater layoutInflater = LayoutInflater.from(mContext);
+        View view = layoutInflater.inflate(R.layout.list_group_item, parent, false);
+        return new GroupHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(GroupAdapter.GroupHolder holder, int position) {
+        //Placeholder text for now
+        holder.mAvatarImageView.setImageResource(R.drawable.ic_account_circle_grey600_48dp);
+        holder.mTitleTextView.setText("Group " + position);
+        holder.mLastMessageTextView.setText("Last message of group " + position);
+    }
+
+    @Override
+    public int getItemCount() {
+        //Arbitrary for now
+        return 10;
+    }
+
+    public static final class GroupHolder extends RecyclerView.ViewHolder {
+        CircleImageView mAvatarImageView;
```

Comment:
I think in the Holder class, it makes more sense to leave out the "m" for member variable